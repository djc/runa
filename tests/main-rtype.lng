def main(name: str, args: array[str]) -> int:
	print(x)
