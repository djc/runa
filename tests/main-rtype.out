tests/main-rtype.lng [1.42]: 'main()' return type should be 'void'
def main(name: str, args: array[str]) -> int:
                                         ^
